README.md

```diff
@@ -7,14 +7,27 @@ Cloudfuse is a fork of [blobfuse2](https://github.com/Azure/azure-storage-fuse),
 Cloudfuse supports clouds with an S3 or Azure interface.
 
 ## Installation
-Download the provided installation packages for your preferred operating system.  -- COMING SOON!
 
-### Windows
+### Windows 
 
 Download and the .exe installer from our latest release [here](https://github.com/Seagate/cloudfuse/releases). Uncheck the "Launch Cloudfuse" upon finishing the installation. Run the GUI separately as admin after the install completes.
 
 Cloudfuse will store the config file in `	C:\Users\{username}\AppData\Roaming`.
 
+### Linux 
+
+#### Debian /Ubuntu:
+
+Download the .deb file from our latest release [here](https://github.com/Seagate/cloudfuse/releases) and run the following command in your terminal: 
+
+sudo apt-get install -f ./path/to/your/.deb

```

Comment:
Instead of `/path/to/your/.deb`, maybe we can assume they're in the same folder where the deb file is.
So it could be `cloudfuse*.deb`, which would work verbatim with copy paste.